security-admin/src/main/resources/conf.dist/ranger-admin-site.xml

```diff
@@ -258,6 +258,10 @@
 		<name>ranger.web.authentication.alt-kerberos.portal</name>
 		<value>https://localhost/sso/authorize</value>
 	</property>
+	<property>
+		<name>ranger.web.authentication.alt-kerberos.portal.logout</name>
+		<value>https://localhost/sso/logout</value>
```

Comment:
this is an example? Port number is missing, and I think ranger UI port isn't 80? I would even keep this empty and LOG.fatal if it's not set in the code...

Reply:
Yes, this is a dummy example. We actually use our own portal's logout link in the deployment. I will fix and keep it empty if makes this clearer.